component/otelcol/exporter/prometheus/internal/convert/convert.go

```diff
@@ -287,7 +286,8 @@ func (conv *Converter) consumeMetric(app storage.Appender, memResource *memorySe
 }
 
 func (conv *Converter) consumeGauge(app storage.Appender, memResource *memorySeries, memScope *memorySeries, m pmetric.Metric) {
-	metricName := prometheus.BuildPromCompliantName(m, "")
+	// TODO: should we make the param addMetricSuffixes configurable? For now we set the default value (true) to keep the same behavior as before
+	metricName := prometheus.BuildCompliantName(m, "", true)
```

Comment:
Should we expose this parameter or we keep it as it was before by setting it to true?

Reply:
Here is the change: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24260/files